docs/_static/mxnet.css

```diff
@@ -606,9 +607,13 @@ div.sphinxsidebar ul ul { margin-left: 15px }
     padding-bottom: 8px;
 }
 
+.section ul ul {
+    display: none
```

Comment:
This broke the documentation on the Installation page. Under minimum requirements, the list of BLAS libraries is now hidden. http://mxnet.io/get_started/setup.html#minimum-requirements is hiding the following lines: https://github.com/dmlc/mxnet/blame/00b5245/docs/get_started/setup.md#L28-L30